crane-cloud/frontend
# fix for fetching  updated apps

# Description

This PR is to display  apps to the user that were damaged by  the old app update feature, it also involves the option the recovery of a URL  for all the apps that were damaged the old update feature. 
Note: the Url cant be regeneated  on the staging cluster apps because it does'nt support ingress  
Also shows number of apps per project.

## Type of change


## Trello Ticket ID

https://trello.com/c/Oy4xXYvC

## How Can This Been Tested?

Pull the branch and try to check out all apps from projects that cant recover their apps any more, and if they are on the staging cluster the urls may not regenerate the url 

## Checklist:


# Screenshots

![settings](https://user-images.githubusercontent.com/69305400/173532863-d6f8183b-95c2-4c94-b03a-4be46eac6c07.png)
